doc/yast-storage-requirements.md

```diff
@@ -0,0 +1,346 @@
+* AI: speak with different parties which requirements they have on automated proposal (Cloud, SES, SAP, ...)
+* AI: Define when dedicated boot partition is needed before merging into the 
+* AI: Find otu if EVMS support for upgrade is necessary (when was it last supported?)
+* AI: Provide specifics for zFCP devices
+* AI: Define requirements on /boot/zipl partition
+* AI: Provide details about specific tools for first sector writing, reasoning for two copies of kernel (why two?) for DASD
+
+
+## Disks and Partitions Simple case:
+
+   * Disk  
+       * Partition table
+           * Partition  
+               * File system  
+                      * File
+                      * File
+                      * File
+                      * ...
+           * Partition  
+               * File system  
+                      * File
+                      * ...
+           * Partition  
+               * ...
+           * ...
+
+## Advanced Cases
+
+This covers various ways to combine disks and/or use advanced
+ways to manage the disk space provided by the disks
+
+###RAID (Redundant Array of Independent Disks)
+
+   * Hardware RAID: mostly automatic and transparent
+       * Behaves like any regular disk
+       * No need for any special handling (or even detection)
+   * Linux Software RAID / BIOS RAID (type dependent on BIOS):  
+       * mdadm RAID: https://en.wikipedia.org/wiki/Mdadm
+       * dmraid (Device Mapper RAID): https://en.wikipedia.org/wiki/Device_mapper ?
+
+BIOS-managed raid is configured in BIOS (and Linux system detects it), Linux softwre RAID must be configured in Linux
+
+RAID levels:
+
+   * **RAID 0**: striping; performance improvement due to parallel disk I/O; less fault tolerance than one disk
+   * **RAID 1**: mirroring; improved fault tolerance, same performance as one disk
+   * **RAID 5**: 3 or more disks with redundancy (for parity blocks): Good fault tolerance, improved performance (parallel I/O)
+   * **RAID 6**: 4 or more disks; more redundancy than RAID 5, thus even better fault tolerance
+   * **RAID 10**: striping + mirroring; combination of RAID 1 and RAID 0
+
+Supported actions:
+
+   * create RAID 0, 1, 5, 6, 10
+   * create partitions on RAID (if there already is a partition table; cannot create a partition table if there is not one already- AI: Arvin - why?)
+
+Not supported by YaST storage - use dedicated tools instead:
+
+   * add another disk (to already created RAID)
+   * remove disk (from RAID)
+   * create BIOS RAID (use BIOS or mdadm/dmraid)
+   * These cases need to be documented properly
+
+Raid and boot partitions
+
+   * AI: Define which Raid configurations are bootable?
+
+### Volume Managers
+
+   * LVM
+   * Btrfs (volume manager features): not yet supported, needs FATE with usecases and technical information
+   * EVMS: legacy; no longer supported, drop
+   * AI: Find out if we need upgrade support of EVMS
```

Comment:
I don't think we need any upgrade support for EVMS since EVMS was already dropped for SLES11. See e.g. fate #305007 or https://bugzilla.suse.com/show_bug.cgi?id=848821#c18.